support-workers/src/main/scala/com/gu/support/workers/lambdas/UpdateSupporterProductData.scala

```diff
@@ -103,7 +103,7 @@ object UpdateSupporterProductData {
                 identityId = user.id,
                 productRatePlanId = productRatePlan.id,
                 productRatePlanName = s"support-workers added ${product.describe}",
-                Some(ContributionAmount(product.amount, product.currency.iso)),
+                None, // We don't send the amount for S+, because it may be discounted
```

Comment:
we don't actually use this even for contributions, but it's been kept "just in case" even after apps don't need it via members-data-api, see https://github.com/guardian/members-data-api/pull/850
The current MDAPI code doesn't use it at all for any product.